chiperific/liters_tracker
# Story validation

Test steps:
1. For both add and edit pages...
2. Try saving with various combinations of forms unfilled
3. It should refuse to submit and highlight necessary forms if the title and/or body is missing

Note: Check the navigation if you refresh on a failed validation page - does it navigate to the correct page?
